### ISSUE TITLE
Reversibility mismatch in SBML and dat file

### ISSUE THREAD

**Issue by y1zhou** [y1zhou/metabolike#25]
Here's the statistics for HumanCyc:

| reactionDirection       | reversible | COUNT(*) |
|:------------------------|-----------:|---------:|
| "left_to_right"         | false      | 871      |
| "physiol_left_to_right" | false      | 1519     |
| "physiol_right_to_left" | false      | 172      |
| "right_to_left"         | false      | 87       |
| "reversible"            | true       | 239      |
| "reversible"            | false      | 17       |
| null                    | true       | 146      |
| "physiol_left_to_right" | true       | 1        |

The `reactionDirection` field comes from reactions.dat, and `reversible` comes from the SBML file. To match with the results shown in HumanCyc:

- When `reactionDirection` is "reversible" and `reversible` is false, the reactions **are reverisble**.
- When `reactionDirection` is missing and `reversible` is true, the direction is **unknown**. 
- When `reactionDirection` is not `reversible` and `reversible` is true, the reaction is **irreversible**.

In summary, the `reversible` property is not reliable and can be dropped.
